packages/cli/generators/extension/templates/src/controllers/README.md

```diff
@@ -0,0 +1,3 @@
+# Controllers
```

Comment:
The `Mixin`, `Providers`, etc.. pages have a great deal of description. Can we add some links to our docs or brief explanations here as well?